test/e2e/Tests/Helpers/HttpHelpers.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Durable.Tests.DotnetIsolatedE2E
+{
+    class HttpHelpers
+    {
+        public static async Task<HttpResponseMessage> InvokeHttpTrigger(string functionName, string queryString = "")
+        {
+            // Basic http request
+            HttpRequestMessage request = GetTestRequest(functionName, queryString);
+            request.Headers.Accept.Add(new MediaTypeWithQualityHeaderValue("text/plain"));
+            return await GetResponseMessage(request);
+        }
+
+        public static async Task<HttpResponseMessage> InvokeHttpTriggerWithBody(string functionName, string body, string mediaType)
+        {
+            HttpRequestMessage request = GetTestRequest(functionName);
+            request.Content = new StringContent(body);
+            request.Content.Headers.ContentType = new MediaTypeHeaderValue(mediaType);
+            request.Headers.Accept.Add(new MediaTypeWithQualityHeaderValue(mediaType));
+            return await GetResponseMessage(request);
+        }
+
+        public static async Task<bool> InvokeHttpTrigger(string functionName, string queryString, HttpStatusCode expectedStatusCode, string expectedMessage, int expectedCode = 0)
+        {
+            string uri = $"{Constants.FunctionsHostUrl}/api/{functionName}{queryString}";
+            using (var request = new HttpRequestMessage(HttpMethod.Get, uri))
+            {
+                request.Headers.Accept.Add(new MediaTypeWithQualityHeaderValue("text/plain"));
+                var response = await GetResponseMessage(request);
+
+                Console.WriteLine(
+                    $"InvokeHttpTrigger: {functionName}{queryString} : {response.StatusCode} : {response.ReasonPhrase}");
```

Comment:
Will console logs be captured in the test output? Normally we write logs in Xunit to `ITestOutputHelper`.

Reply:
Deleting this whole method as it is never used and - IMO - we should rely on the Task<HttpResponseMessage> method and validate responses in tests or methods defined in test classes